fpna/Utils/ExtentionMethods.ipynb

```diff
@@ -59,8 +81,11 @@
             "cell_type": "markdown",
             "source": [
                 "## Compute Compounded Cashflow",
+                "\nThis method provides a geometrically increasing or decreasing cash flow.",
                 "\n",
-                "\nTODO: Add Short description"
+                "\nThe **amount** of the cash flow in the first period is equal to \"**Initial Value**\" * (1 + **Value**(t)). In subsequent periods, the cash flow is equal to the previous cash flow multiplied by (1 + **Value**(t)).",
```

Comment:
why (1+Value(t)) and not just Value(t)?
if you want the cashflow to go to:
- 0, you need to provide -1 (0 would be more intuitive)
- -previousValue, you need to provide -2 (-1 would be more intuitive)

Reply:
prefer to keep 1+  Value, to allow providing growth rate (e.g. interest rate values, will provide a growth of an amount over time)